packages/logger/lib/logger.js

```diff
@@ -82,6 +83,34 @@ const logLevelToTransportMethodMap = {
 	warn: { logLevel: 'warn', isDeprecated: false }
 };
 
+/**
+ * Whether log prettification is available. This is based
+ * on two things: the pino-pretty module being installed
+ * in the application, and the `NODE_ENV` environment
+ * variable being undefined or "development".
+ *
+ * @type {boolean}
+ */
+const PRETTIFICATION_AVAILABLE = (() => {
+	try {
+		// We have to `require` here rather than `require.resolve`
+		// which is less than ideal but otherwise this is actually
+		// impossible to test with Jest. Both technically do the
+		// same file system work though, and it's only done once
+		// when the module first loads. It's also safe to ts-ignore
+		// this one because it's never actually used directly.
+		// @ts-ignore
+		require('pino-pretty');
```

Comment:
**thought**: `pino-pretty` is the de facto standard prettifier, but there are others. hardcoding this precludes apps from choosing a different one. but that's probably fine

Reply:
Yeah I think that's a different configuration thing later. I'd rather start off being opinionated and introduce more variance as and when people need it 🙂